### PR TITLE
[DXP-878] Add config for generic relay-service

### PR DESCRIPTION
**Pull request by andrzejmikolajczakds**

### CODE REVIEW

File: set-up-page-generation-tutorial/mesh.yml [hunk 1]
```diff
@@ -14,26 +14,53 @@ sources:
...
+        topic: inboxes/web-resources
+    outgoing:
+      relayed-messages:
         topic: outboxes/web-resources
+    environment:
+      MP_MESSAGING_INCOMING_MESSAGES_SCHEMA: web-resource-schema
+      MP_MESSAGING_OUTGOING_RELAYED-MESSAGES_SCHEMA: web-resource-schema
 
   rendering-engine-processing-service:
     image: rendering-engine-processing-service
```
Comment (andrzejmikolajczakds) [inline set-up-page-generation-tutorial/mesh.yml:66]: Container of this service fails to run

**Review by stan-r-ds — APPROVED** [review]

File: build-data-aggregation-tutorial/mesh.yml [hunk 2]
```diff
@@ -11,14 +11,17 @@ sources:
       - "rendering-contexts"
 
 processing:
-  relay:
+  relay-pages:
```
Comment (pawel-boguski-ds) [inline build-data-aggregation-tutorial/mesh.yml:14]: maybne pages-relay - as this is name of the service this form would be better

File: create-simple-website-tutorial/mesh.yml [hunk 3]
```diff
@@ -10,22 +10,41 @@ sources:
       - "web-resources"
 
 processing:
-  relay:
+  relay-pages:
```
Comment (pawel-boguski-ds) [inline create-simple-website-tutorial/mesh.yml:13]: maybne pages-relay - as this is name of the service this form would be better

Comment (pawel-boguski-ds) [reply create-simple-website-tutorial/mesh.yml:13]: same with other names

File: set-up-page-generation-tutorial/mesh.yml (hunk 1 above)
Comment (pawel-boguski-ds) [reply set-up-page-generation-tutorial/mesh.yml:66]: remove outgoing as the serivce was changed:
 compositions:
        topic: inboxes/compositions

**Review by pawel-boguski-ds — APPROVED** [review]

**Pull request merged**